src/io.ts

```diff
@@ -98,8 +99,15 @@ export const startIOServer = (httpServer: ServerType) => {
         // no more questions, so it is the end of the quiz
         if (schema == null) {
           console.log('quiz done');
-          // TODO: include leaderboard results with event
-          io.to(roomId).emit(WS_EVENTS.QUIZ_COMPLETED);
+          // sort leaderboard by scores and get top 3
+          const leaderboard = Array.from(playerScores.entries()).sort((a, b) => b[1] - a[1]).slice(0, 3);
+          const playerScore = playerScores.get(playerId);
```

Comment:
[NIT] `playerScore` will have a type of `number | undefined`, can you make such that it is only number. For example:
```javascript
const playerScore = playerScores.get(playerId) ?? 0;
```